src/Common/src/TypeSystem/Common/TypeHashingAlgorithms.cs

```diff
@@ -208,6 +208,18 @@ public static int ComputeGenericInstanceHashCode<ARG>(int genericDefinitionHashC
             return (hashcode + _rotl(hashcode, 15));
         }
 
+        public static int ComputeMethodSignatureHashCode<ARG>(int returnTypeHashCode, ARG[] parameters)
+        {
+            // TODO: Do we need calling conventions in the mix too?
```

Comment:
@davidwrighton The problem with this is that we don't have a common enum type to specify them.

I could make TypeHashingAlgorithms define it's own and have everyone convert between what they use and what TypeHashingAlgorithms use. Any other ideas?